h2/src/main/org/h2/mode/PgCatalogTable.java

```diff
@@ -579,8 +596,12 @@ public PgCatalogTable(Schema schema, int id, int type) {
                         ValueInteger.get((int) pgType[2]),
                         // TYPTYPE
                         pgType[3],
+                        // TYPDELIM
+                        ",",
                         // TYPRELID
                         ValueInteger.get(0),
+                        // TYPELEM
+                        ValueInteger.get(0),
```

Comment:
`19/name` has 18 (single character, special type) and `22/int2vector` has `PG_TYPE_INT2` in the real PostgreSQL.

For 18 special fake row `18/char` may be needed.

I also don't see `0/null` in the real PostgreSQL, perhaps it should be removed.

`typtype=b` for all these special types, but currently H2 returns `b` only for `regproc`.

Reply:
In fact oid for `regproc` in PostgreSQL is 24 but not 2205. May be special reason in H2's history.

Reply:
It was introduced in 18cfcf6c5e51f05ac4eca957cb619e4eeb020be6, see also https://code.google.com/archive/p/h2database/issues/481

Actually It looks like copy-paste error from PosgreSQL query results, `regclass` is 2205, but `regproc` is 24. I'm not sure what should be here, maybe both of them.